pages/register.js

```diff
@@ -15,7 +15,7 @@ const Register = () => {
   useEffect(() => {
     
     if (!isAuthenticated()) {
-      authenticate('https://reflectionsprojections.org/auth')
+      //authenticate('https://reflectionsprojections.org/auth')
```

Comment:
Why is this commented out? Was it for testing purposes?

Reply:
Yep, it should be uncommented now.